migration/v0.36/apply/50-velero-crds-upgrade.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+      log_info "  - applying the Velero CRDs on sc"
+      kubectl_do sc apply --server-side --force-conflicts -f "${ROOT}"/helmfile.d/upstream/vmware-tanzu/velero/crds
+
+      helmfile_upgrade sc app=velero
+    fi
+    if [[ "${CK8S_CLUSTER}" =~ ^(wc|both)$ ]]; then
+      log_info "  - applying the Velero CRDs on wc"
+      kubectl_do wc apply --server-side --force-conflicts -f "${ROOT}"/helmfile.d/upstream/vmware-tanzu/velero/crds
+
+      helmfile_upgrade wc app=velero
```

Comment:
The file seem to indicate that it upgrades the CRDs only but this will also upgrade Velero itself as well right?

Reply:
yes, upgrade both

Reply:
As a nit-pick this file could be renamed to "50-velero-upgrade.sh" to make it more clear that it is upgrading all of velero. But it is ok if you skip that.